### ISSUE TITLE
File not found with an addon

### ISSUE THREAD

**Issue by mfeckie** [embroider-build/embroider#672]
:wave: I have an addon (https://github.com/mfeckie/fast-sass) which we use for SCSS compilation.

To support a pod / modular CSS approach, we have an option `autoIncludeComponentCSS` which essentially collections the names of all the SCSS files in `app/components` and builds it into imports.

```scss
@import 'app/components/app-header/style';
@import 'app/components/login-page/style';
@import 'app/components/password-reset-request-page/style';
@import 'app/components/password-reset-link-page/style';
```

This gets output into `app/styles/pod-styles.scsss` and is generated at build time, prior to the SASS compiler running.

It imported via `app.scss`

```scss
@import 'pod-styles';
```

When I try to make the addon compatible with `embroider` it can't find the autogenerated file any more

```
 Build Error (SassCompiler) in /tmp/broccoli-3035hQcrKh3GK0Aa/out-121-funnel/app.scss:2:9

Can't find stylesheet to import.
  ╷
2 │ @import "pod-styles";
  │         ^^^^^^^^^^^^
  ╵
  /tmp/broccoli-3035hQcrKh3GK0Aa/out-121-funnel/app.scss 2:9  root stylesheet
```

Reference branch here https://github.com/mfeckie/fast-sass/pull/2

**Comment by roberkules** [comment]
@mfeckie just wanted to ask if you came across a fix... I'm having (I guess) the same issue (but I'm not using embroider).
the build worked a few weeks ago, there were no changes and now fails with same error as yours

```Sass Syntax Error (SassCompiler) in 0/components/contact-details/styles.scss:1:1

Error: File to import not found or unreadable: appcard-web-ui/variables.
        on line 1 of tmp/sass_compiler-input_base_path-YG8cLdL0.tmp/0/components/contact-details/styles.scss
        from line 1 of tmp/sass_compiler-input_base_path-YG8cLdL0.tmp/0/pod-styles.scss
        from line 1 of tmp/sass_compiler-input_base_path-YG8cLdL0.tmp/0/addon.scss
>> @import "appcard-web-ui/variables";```

**Comment by mfeckie** [comment]
@roberkules This is specifically related to embroider.  The addon works fine without.

What library are you using for the sass inclusion?

**Comment by roberkules** [comment]
@mfeckie I'm using

```
{
  "ember-source": "~3.17.0",
  "ember-cli": "~3.14.0",
  "ember-cli-sass": "~10.0.0",
  "node-sass": "4.11.0",
  "ember-component-css": "~0.6.3" // also tried "0.7.4"
}
```

**Comment by mfeckie** [comment]
@roberkules OK, sorry, I'm only familiar with `fast-sass` (because I wrote it 😂 ).  It doesn't _sound_ like the same issue as you said you're not using embroider.

There does seem to be some discussion about it over here https://github.com/salsify/ember-css-modules/issues/140 but again in the context of embroider

**Comment by rwjblue** [comment]
Closing (we think it is fixed by #674), should be in the next release.
